### PR TITLE
[fix] Updated `createSelect` to add DOM Element Support

### LINKED ISSUE #4394: createSelect does not handle existing dom elements
#### Most appropriate sub-area of p5.js?

- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [x] Other (dom)

#### Details about the bug: 

- p5.js version: 1.0.0
- Web browser and version: Chrome 80.0.3987.122
- Operating System: Manjaro 19.0.2 Kyria (Linux 4.19.108-1-MANJARO)
- Steps to reproduce this:

As per the documentation, [`createSelect`](https://p5js.org/reference/#/p5/createSelect) should be able to take an existing DOM select element and return a `p5.Element` of select type. But the current implementation does not handle such a case and throws an Error. It only handles either a `p5.Element` or creates a new one. Is it documentated wrong or implemented wrong ?

### PR DESCRIPTION
**Pull request by fenilgandhi**
Resolves #4394 

 Changes:
I am sure as what to look up to, as a source of truth but according to my understanding, i have done the following changes.
##### Documentation
- specified methods on seperate lines
- updated their description to match be more explanatory.
- split descriptions of method behaving differently on different parameters.

##### Implementation
- Replaced `var` with `let`.
- Added handling of DOM select, it will wrap up the DOM select in a `p5.Element` and return it.
- Handled cases for validating parameter in methods.

Once the changes in `createSelect` are finalised, i will update the unit tests also.

#### PR Checklist
- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [ ] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/master/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/master/contributor_docs#unit-tests

### CODE REVIEW

File: src/dom/dom.js [hunk 1]
```diff
@@ -737,16 +743,18 @@ p5.prototype.createSelect = function() {
...
+        if (this.elt[i].value.toString() === value) {
           this.elt[i].disabled = true;
         }
       }
       return this;
-    } else if (arguments.length === 0) {
-      this.elt.disabled = true;
-      return this;
+    } else {
+      // TODO : Need a default behaviour here.
```
Comment (fenilgandhi) [inline src/dom/dom.js]: What should be done here ?

Comment (lmccart) [reply src/dom/dom.js]: I think we should switch this to remove the undefined check, first check if it's a string, then put the lines 747, 748 under "else"

Comment (fenilgandhi) [reply src/dom/dom.js]: Ok thanks.

**Comment by lmccart** [comment]
looks great, thank you!

**Pull request merged**